.github/ISSUE_TEMPLATE/bug_report.yml

```diff
@@ -1,114 +1,133 @@
+      options:
+        - label: I am using the latest version (check for updates in Settings)
+          required: true
+        - label: I have searched the [online documentation](https://aka.ms/PowerToysOverview)
+          required: true
+        - label: I have searched in [open issues](https://github.com/microsoft/PowerToys/issues) (remember to try different search terms)
+          required: true
+        - label: I have searched in [closed issues](https://github.com/microsoft/PowerToys/issues?q=is%3Aissue+is%3Aclosed) (there might be a good reason)
+          required: true
+
+  - type: input
+    id: version
+    validations:
+      required: true
+    attributes:
+      label: Microsoft PowerToys version
+      description: Hover over system tray icon or look at Settings
+      placeholder: 0.80.0
+
+  - type: dropdown
+    id: installation
+    validations:
+      required: true
+    attributes:
+      label: Installation method
+      options:
+        - GitHub
+        - PowerToys auto-update
+        - Microsoft Store
+        - WinGet
+        - Chocolatey
+        - Scoop
+        - Dev build in Visual Studio
+        - Other (please specify in 'Steps to reproduce')
 
-- type: dropdown
-  attributes:
-    label: Installation method
-    description: How / Where was PowerToys installed from?
-    multiple: true
-    options:
-      - GitHub
-      - PowerToys auto-update
-      - Microsoft Store
-      - WinGet
-      - Chocolatey
-      - Scoop
-      - Dev build in Visual Studio
-      - Other (please specify in "Steps to Reproduce")
-  validations:
-    required: true
+  - type: dropdown
+    id: area
+    validations:
+      required: true
+    attributes:
+      label: Area(s) with issue
+      description: Multiple selections is possible
+      multiple: true
+      options:
+        - Always on Top
+        - Awake
+        - ColorPicker
+        - Command not found
+        - Crop and Lock
+        - Environment Variables
+        - FancyZones
+        - FancyZones Editor
+        - File Locksmith
+        - "File Explorer: Preview Pane"
+        - "File Explorer: Thumbnail preview"
+        - Hosts File Editor
+        - Image Resizer
+        - Installer
+        - Keyboard Manager
+        - Mouse Utilities
+        - Mouse Without Borders
+        - Paste as Plain Text
+        - Peek
+        - PowerRename
+        - PowerToys Run
+        - Quick Accent
+        - Registry Preview
+        - Screen ruler
+        - Settings
+        - Shortcut Guide
+        - System tray interaction
+        - TextExtractor
+        - Video Conference Mute
+        - Welcome / PowerToys Tour window
+        - General / Other
 
-- type: dropdown
-  attributes:
-    label: Running as admin
-    description: Are you running PowerToys as Admin?
-    options:
-      - "Yes"
-      - "No"
+  - type: dropdown
+    attributes:
+      label: Running as Administrator?
+      options:
+        - "Yes"
+        - "No"
```

Comment:
This has to be marked as required, otherwise you can choose "None"

![image](https://github.com/microsoft/PowerToys/assets/58633848/4a6cb79d-51ab-4923-b431-5bf4bc13e951)

Reply:
"none" means no selection, it's not a literal option

Reply:
> "none" means no selection, it's not a literal option

Yes, but I think if you mark it as required this option will be unavailable.